moveit_planners/ompl/ompl_interface/src/model_based_planning_context.cpp

```diff
@@ -89,7 +89,7 @@ void ompl_interface::ModelBasedPlanningContext::setProjectionEvaluator(const std
 {
   if (!spec_.state_space_)
   {
-    logError("No state space is configured yet");
+    ROS_ERROR_NAMED("model_based_planning_context", "No state space is configured yet");
```

Comment:
Only `ompl`?

Reply:
No, the ompl package is too big and confusing to use the top-level namespace here.

Reply:
And as you stated above, the package name is something like ompl already...